lib/full-site-editing/template-parts.php

```diff
@@ -266,3 +266,83 @@ function gutenberg_block_header_area() {
 function gutenberg_block_footer_area() {
 	gutenberg_block_template_part( 'footer' );
 }
+
+register_block_pattern(
```

Comment:
These are just copies of tt1-blocks template parts thrown in here as patterns for testing purposes.

A ❓ question to answer before shipping is - should we register any template-part block patterns in Gutenberg (or leave that to themes/plugins)?  And if so what should those patterns be?

Reply:
-cc @kjellr @mtias

Reply:
It largely depends on what comes up from the work on the navigation block and patterns for header areas. We would include some patterns for headers once those are ready.

Reply:
This question is coming up for a lot of blocks. 😄 I've opened https://github.com/WordPress/gutenberg/issues/31231 for us to start exploring some of these patterns. 

It strikes me that when we combine template part patterns with all of our query block patterns, we're basically including a handful of mix-and-match themes for folks to choose from. 🤔